chain/force-upgrade/src/lib.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) The Starcoin Core Contributors
+// SPDX-License-Identifier: Apache-2
+
+use anyhow::format_err;
+use include_dir::{include_dir, Dir};
+use starcoin_types::account::DEFAULT_EXPIRATION_TIME;
+use starcoin_types::identifier::Identifier;
+use starcoin_types::language_storage::ModuleId;
+use starcoin_types::{
+    account::{Account, DEFAULT_MAX_GAS_AMOUNT},
+    transaction::SignedUserTransaction,
+};
+use starcoin_vm_types::account_config::core_code_address;
+use starcoin_vm_types::transaction::ScriptFunction;
+use starcoin_vm_types::{
+    account_config::STC_TOKEN_CODE_STR,
+    genesis_config::ChainId,
+    transaction::{Package, RawUserTransaction, TransactionPayload},
+};
+
+pub const FORCE_UPGRADE_PACKAGE: Dir = include_dir!("../../vm/stdlib/compiled/12/11-12");
```

Comment:
https://docs.rs/include_dir/latest/include_dir/ avoid using relative path. @simonjiao @welbon .Maybe you can use COMPILED_MOVE_CODE_DIR